src/routing/identity/login.tsx

```diff
@@ -13,7 +13,7 @@ export const LoginRoute: FC = () => {
   return (
     <Switch>
       <NotAuthenticatedRoute exact path={`${path}`}>
-        <LoginPage flow={flow} />;
+        <LoginPage flow={flow} />
```

Comment:
Could start adopting the entities/actions pattern. Not critical, just mentioning it

Reply:
Not now.